understanding/20/pause-stop-hide.html

```diff
@@ -110,58 +88,58 @@ <h2>Intent of Pause, Stop, Hide</h2>
          moves the focus away) would not be considered a "mechanism for the user to pause"
          because it makes the page unusable in the process and would not meet this SC.
       </p>
-      
-      <p>It is important to note that the terms "blinking" and "flashing" can sometimes refer
-         to the same content.
-      </p>
-      
+
+      <div class="note">
+         <p>This Success Criterion is specifically concerned with moving, blinking, scrolling, and
+            auto-updating visual content. For audio content that starts automatically, refer to <a href="audio-control">1.4.2 Audio Control</a>.</p>
+         <p>Moving, blinking, scrolling content that starts automatically not as a result of explicit *activation* (such as activating a button),
+            but because of a more general user interaction (such as focusing/hovering over an element, or scrolling the page), and doesn't provide
+            a way to Pause, Stop, or Hide, will fail this Criterion, and potentially
+            <a href="animation=from=interaction">2.3.3 Animation from Interaction</a>.</p>
+      </div>
+
+      <p>It is important to note that the terms "blinking" and "flashing" can sometimes refer to the same content.</p>
+
       <ul>
-         
          <li>"Blinking" refers to content that causes a distraction problem. Blinking can be allowed
             for a short time as long as it stops (or can be stopped)
          </li>
-         
          <li>"Flashing" refers to content that can trigger a seizure (if it is more than 3 per
             second and large and bright enough). This cannot be allowed even for a second or it
             could cause a seizure. And turning the flash off is also not an option since the seizure
             could occur faster than most users could turn it off.
          </li>
-         
          <li>Blinking usually does not occur at speeds of 3 per second or more, but it can. If
             blinking occurs faster than 3 per second, it would also be considered a flash.
          </li>
-         
       </ul>
-      
-      
+
+      <div class="note">
+         <p>"Flashing" content that starts automatically will fail this Success Criterion, as well as <a href="three-flashes">2.3.2 Three Flashes</a>
```

Comment:
Not sure cross-referencing 2.3.2 / 2.3.1 here is that helpful - it might then need to reiterate the exceptions of 2.2.2 - because if flashes take 4 sec they'd fail 2.3.2 but would pass 2.2.2?

Reply:
I softened/generalised this - rather than trying to repeat the failure conditions, now just says that content will need to be evaluated against those SCs